pages/react-famewall.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-famewall';
```

Comment:
Why is this file required? I checked online and it seems that famewall is built in TS and hence is shipped with types.